Mage/Mage.cs

```diff
@@ -49,8 +52,11 @@ public class Mage : Singleton<Mage> {
 		messageStream = new MessageStream();
 		archivist = new Archivist();
 
-		_consoleWriter = new ConsoleWriter ();
+		_consoleWriter = new ConsoleWriter();
 		_logger = logger("mage");
+
+		// create a shared cookie container
+		cookies = new CookieContainer();
```

Comment:
I guess for now this is the only place to put cross-transport stuff like this? It feels like the wrong place, but perhaps there is no better place for it at the moment?